src/material-experimental/mdc-helpers/_focus-indicators.scss

```diff
@@ -58,6 +58,15 @@
     margin: 5px;
   }
 
+  // These components have to set `border-radius: 50%` in order to support density scaling
+  // which will clip a square focus indicator so we have to turn it into a circle.
+  .mat-mdc-radio-button,
```

Comment:
Yeah, I noticed this as well. Regardless of the clipping, it looks better to render circular focus indicators on these controls. I'd go a step further and say we should render circular focus indicators on all components with circular ripples (i.e. mat-icon-button, mat-checkbox, mat-radio-button, mat-slide-toggle, date picker cells, etc) instead of just spot-fixing these two components. WDYT?

Reply:
I agree that the circular indicators look better, but I kind of like that the rectangular ones mimic the native browser focus outline. I don't feel strongly about it though.

Reply:
Sorry for the delayed response on this. I agree that it's nice that the rectangular ones somewhat match Chrome's default focus ring.

I still think the strong focus indicators should either be completely component-agnostic (and thus rectangular on every component), or match the shape of the component. It feels half-baked to have focus indicators circular on two components to work around some technical issues. Curious what @jelbourn thinks.

Reply:
I agree with Zack, I think if we're going to start making some of the indicators circular, we should just do it for all of the circular-ripple components

Reply:
I have no strong opinion; as long as they don't look broken it's fine with me

Reply:
I'm fine with merging this PR to fix the ones that look broken and just filing an issue to deal with the other ones that should be circular